android/runtime/common/src/js/titanium.js

```diff
@@ -16,10 +16,10 @@ var tiBinding = kroll.binding('Titanium'),
 	url = require('url'),
```

Comment:
<!--
  0 failure: 
  1 warning:  android/runtime/c...
  
  
  DangerID: danger-id-default;
  File: android/runtime/common/src/js/titanium.js;
  Line: 16;
-->

- :warning: android/runtime/common/src/js/titanium.js line 16 – 'url' is assigned a value but never used. (no-unused-vars)